.github/workflows/ci.yml

```diff
@@ -0,0 +1,90 @@
+name: CI
+
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+
+jobs:
+  qa:
+    name: QA
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        toxenv:
+          - checkqa
+          - readme
+    steps:
+      - uses: actions/checkout@master
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+      - name: Set PY
+        run: echo "::set-env name=PY::$(python -VV | sha256sum | cut -d' ' -f1)"
+      - uses: actions/cache@v1
+        with:
+          path: ~/.cache/pre-commit
+          key: pre-commit|${{ env.PY }}|${{ hashFiles('.pre-commit-config.yaml') }}
+      - name: Install tox
+        run: pip install tox
+      - name: Test ${{ matrix.toxenv }}
+        run: tox -e ${{ matrix.toxenv }}
+
+  cpython:
+    name: Test
```

Comment:
```suggestion
    name: ${{ matrix.os }}/${{ matrix.python-version }}/${{ matrix.pip-version }}
```

Reply:
<img width="435" alt="image" src="https://user-images.githubusercontent.com/7377671/80891723-16284900-8cf0-11ea-897c-c414ed0e412d.png">

Don't you like this notation?

Reply:
Unfortunately, the problem with job names in GitHub Actions is that some places where it's rendered are very narrow so job names must be short to fit.

Reply:
<img width="249" alt="Screen Shot 2020-05-02 at 10 44 40 PM" src="https://user-images.githubusercontent.com/578543/80891777-9e91f400-8cc6-11ea-943e-609c4c3f83ee.png">

Reply:
I often use emojis and short forms trying to fight this.

Reply:
Could you attach the full screen of the page?

Reply:
I understand where this is coming from but no, it's not because of narrow window size, there's no responsiveness — the width is fixed.
It's here, see for yourself: https://github.com/tox-dev/azure-pipelines-template/blob/master/run-tox-env.yml#L133 (scroll down the list).
<img width="1280" alt="Screen Shot 2020-05-02 at 11 02 17 PM" src="https://user-images.githubusercontent.com/578543/80892269-fe899a00-8cc8-11ea-8814-074301904a04.png">

Reply:
Wrong link I guess, but I get that. Yeah, that page pretty fixed.

Reply:
Addressed in 988dfbc.